src/hooks/useUsers.ts

```diff
@@ -0,0 +1,61 @@
+import { useMemo } from "react";
+import { useFirestoreConnect } from "react-redux-firebase";
+
+import { usersSelector, usersByIdSelector } from "utils/selectors";
+import { WithId } from "utils/id";
+
+import { useSelector } from "hooks/useSelector";
+import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
+import { useVenueId } from "hooks/useVenueId";
+
+import { User } from "types/User";
+
+const useConnectUsers = () => {
+  useFirestoreConnect("users");
+};
```

Comment:
Since my intent was to unify the way we interact with data, I think `useConnects` must be used in related hooks only. If you have any use cases, where we would need to explicitly use `useConnectUsers` outside of `users` hooks, I'd love to hear about that

Reply:
I think our ideal situation in the main app is that we would never need to explicitly invoke `useConnects`, as they should be automagically handled within our other custom hooks. So it's probably fine not to export this.

Sometimes I tend to opt to exporting things 'just in case', and/or to make writing tests easier, but I don't think either of those situations really matter all that much in this instance, so I think this is probably fine to leave as you had it.